presidio-api/cmd/presidio-api/docs/swagger.yaml

```diff
@@ -0,0 +1,1890 @@
+      maskingCharacter:
+        description: the masking char
+        type: string
+        x-go-name: MaskingCharacter
+    title: Mask the given value with the selected characters.
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  Pattern:
+    description: |-
+      Pattern represents one regex (pattern) that is able to detect a certain
+      entity and how confident the result is
+    properties:
+      name:
+        description: The name of the pattern, unique
+        type: string
+        x-go-name: Name
+      regex:
+        description: The Regex used by this recognizer
+        type: string
+        x-go-name: Regex
+      score:
+        description: The confidence of the result
+        format: float
+        type: number
+        x-go-name: Score
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  PatternRecognizer:
+    description: |-
+      PatternRecognizer represents a recognizer which has the ability to detect
+      entities using one or more Patterns
+    properties:
+      blacklist:
+        description: |-
+          A list of words that are considered PII and should always be detected
+          e.g. ["Mr", "Mrs", "account"]
+        items:
+          type: string
+        type: array
+        x-go-name: Blacklist
+      contextPhrases:
+        description: |-
+          A list of strings that if found in the surroundings of the entity, increase the score of the result
+          e.g. ["name", "address", ]
+        items:
+          type: string
+        type: array
+        x-go-name: ContextPhrases
+      entity:
+        description: The entity name which this recognizer can detect
+        type: string
+        x-go-name: Entity
+      language:
+        description: The supported language code, in ISO-639 format, https://en.wikipedia.org/wiki/ISO_639-1
+        type: string
+        x-go-name: Language
+      name:
+        description: The name of the recognizer, unique
+        type: string
+        x-go-name: Name
+      patterns:
+        description: List of supported patterns by this recognizer
+        items:
+          $ref: '#/definitions/Pattern'
+        type: array
+        x-go-name: Patterns
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  RecognizerInsertOrUpdateRequest:
+    description: |-
+      RecognizerInsertOrUpdateRequest represents the request for
+      inserting a new or updating an existing pattern recognizer
+    properties:
+      value:
+        $ref: '#/definitions/PatternRecognizer'
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  RecognizersGetResponse:
```

Comment:
There are various types in the form <Noun><Verb>Request and <Noun><Verb>Response which is counter intuitive. A better form would be <Verb><Noun>Request/Response.
Examples: RecognizerInsertOrUpdateRequest, RecognizersGetResponse

Reply:
I made naming conventions in the swagger side, but we also use imported objects from the the [presidio-genproto](https://github.com/Microsoft/presidio-genproto) repository. 
The example: RecognizersGetResponse is imported, so the change must be done in the other repository.

Reply:
Opened an issue in the presidio-genproto repo.
https://github.com/microsoft/presidio-genproto/issues/31